miqa/core/models/group.py

```diff
@@ -0,0 +1,17 @@
+from uuid import uuid4
+from django.db import models
+
+
+class Group(models.Model):
+    """Models a generic group of items
+
+        Allows one to reference a specific grouping of items.
+
+        It is currently used by Project to select a collection of Artifacts.
+    """
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    name = models.CharField(max_length=255, blank=False)
+    description = models.TextField()
```

Comment:
```suggestion
    name = models.CharField(max_length=255)
    description = models.TextField(blank=True)
```

`blank` defaults to `False`. (I assumed you meant to allow the description to be blank, but maybe not?)

Reply:
Correct, thanks.